docs/data/material/customization/how-to-customize/how-to-customize.md

```diff
@@ -135,26 +130,20 @@ You can rely on the following [global class names](/styles/advanced/#with-materi
 
 ## 2. Reusable style overrides
 
-If you find that you need the same overrides in multiple places across your application, you can use the [`styled()`](/system/styled/) utility to create a reusable component:
+If you find that you need to reuse the same overrides in different locations across your application, you can use the [`styled()`](/system/styled/) utility to create a reusable component:
 
 {{"demo": "StyledCustomization.js", "defaultCodeOpen": true}}
 
-With it, you have access to all of a component's props to dynamically style the component.
-
-## 3. Dynamic variation
+### Dynamic overrides
 
-In the previous section, we learned how to override the style of a MUI component.
-Now, let's see how we can make these overrides dynamic.
-Here are four alternatives; each has its pros and cons.
+Using the `styled()` utility above allows you to add dynamic styles based on the component's props. Here are two ways you can do that:
 
-### Dynamic CSS
+#### Dynamic CSS
 
-Using the `styled()` utility offers a simple way for adding dynamic styles based on props.
+> ⚠️ Note that if you are using TypeScript you will need to update the prop's types of the new component.
```

Comment:
```suggestion
> ⚠️ Note that if you are using TypeScript you will need to update the prop types of the new component.
```

Reply:
I feel like we should move this back right before the demo showing how it should be altered in TypeScript.

Reply:
Before or after? I haven't altered its position.